packages/memory-storage/src/memory-storage.ts

```diff
@@ -26,6 +26,14 @@ export interface MemoryStorageOptions {
      * @default process.env.DEBUG?.includes('*') ?? process.env.DEBUG?.includes('crawlee:memory-storage') ?? false
      */
     writeMetadata?: boolean;
+
+    /**
+     * Whether the memory storage should also write its stored content to the disk.
+     *
+     * You can also disable this by setting the `CRAWLEE_NO_WRITE_TO_DISK` environment variable.
```

Comment:
this is a bit weird, the property is called `writeFilesToDisk`, not `noWriteFilesToDisk`. we support false values on env vars, i would go that way instead.

also it should go through the configuration class, so you can control this via `crawlee.json` too

Reply:
also i'd say the name should be more generic (especially if it should be part of the config class). maybe just `persistentStorage: true/false`?